backend/src/openarchiefbeheer/destruction/api/viewsets.py

```diff
@@ -488,9 +488,12 @@ def get_queryset(self):
             DestructionListItem.objects.all()
             .select_related("zaak")
             .annotate(
+                last_review_comment=Subquery(
```

Comment:
Would it be an idea to expose this in the manager (like annotate_user_permissions) so it can be used in different places as well. And would it be possible/sensible (did not check) to do the annotate in the serializer where it's used (so no 2 parts reasons about the same annotate)?

Reply:
I don't think we are currently using this in other places :thinking: but if at some point we do, then it makes sense to make it reusable!
I also added the annotate to the serialiser in case the query set has not been annotated yet, but then you do it per item instead of doing it once when you get the whole queryset. This is why I did it also in the view.